packages/core/src/config/parse.ts

```diff
@@ -1022,7 +1039,8 @@ export const parseAndValidateConstructorArgs = (
   referenceName: string,
   abi: Array<Fragment>,
   cre: ChugSplashRuntimeEnvironment
-): ParsedConfigVariables => {
+): { args: ParsedConfigVariables; valid: boolean } => {
+  let validArguments = true
```

Comment:
See comment below for more context on why I'm returning a valid boolean again for this function.